app/assets/stylesheets/application.sass.scss

```diff
@@ -35,3 +35,8 @@ $govuk-assets-path: "/";
 .linting-results-table td {
   padding: 10px !important;
 }
+
+// Ensures that the background colours are shown when printing or saving
+@media print {
+  body {-webkit-print-color-adjust: exact;}
```

Comment:
[suggestion]

I'd suggest adding a `.no-print` class that you can use to identify parts of the page that you don't want to appear in print output. Buttons for example.
```css
.no-print {
  display: none !important;
}

Reply:
Good shout, added.